mmv1/third_party/terraform/services/storagetransfer/resource_storage_transfer_job_test.go

```diff
@@ -381,6 +389,69 @@ resource "google_storage_transfer_job" "transfer_job" {
 `, project, dataSourceBucketName, project, dataSinkBucketName, project, transferJobDescription, project)
 }
 
+func testAccStorageTransferJob_eventStreamConfig(project string, dataSourceBucketName string, dataSinkBucketName string, transferJobDescription string) string {
+	return fmt.Sprintf(`
+data "google_storage_transfer_project_service_account" "default" {
+  project = "%s"
+}
+
+resource "google_storage_bucket" "data_source" {
+  name          = "%s"
+  project       = "%s"
+  location      = "US"
+  force_destroy = true
+  uniform_bucket_level_access = true
+}
+
+resource "google_storage_bucket_iam_member" "data_source" {
+  bucket = google_storage_bucket.data_source.name
+  role   = "roles/storage.admin"
+  member = "serviceAccount:${data.google_storage_transfer_project_service_account.default.email}"
+}
```

Comment:
After I did some manual tests with a version of this config that provisions a `google_pubsub_subscription` resource, I found that you also need to make an IAM membership that gives the storage transfer service account permissions to pull messages from pubsub subscriptions. Just a heads up!

This is the type of error you see otherwise:

> Error: googleapi: Error 400: project-123456789@storage-transfer-service.iam.gserviceaccount.com does not have the required pubsub.subscriptions.consume permission on subscription projects/PROJECT/subscriptions/example-subscription., failedPrecondition

Reply:
this makes sense. I was able to fix this with adding 

```hcl
resource "google_pubsub_subscription_iam_member" "editor" {
  subscription = "your-subscription-name"
  role         = "roles/editor"
  member       = "user:jane@example.com"
}
```

Thanks for the help!